public/styles/components/_accordion.scss

```diff
@@ -83,6 +83,14 @@
     max-height: 0;
     overflow: hidden;
     transition: max-height 0.2s ease;
+
+    &.bsds-accordion-collapsed {
```

Comment:
thoughts on bsds-accordion-panel-collapsed/expanded? now that i'm seeing it in context, i'm leaning that way. but it's long.

Reply:
It's truer to what it is with '-panel-'